From 0c42b66af9b293ab23008edc0aef465ab432fd93 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 29 Jul 2020 16:55:39 +0800
Subject: [PATCH] Test case2-E for Add same library instance

Description: Add same 'BaseMemoryLib' library instance (BaseMemoryLib|MdePkg/Library/BaseMemoryLibRepStr/BaseMemoryLibRepStr.inf) for 'DXE_CORE' class.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/LibraryClass.dsc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TestPkg/LibraryClass.dsc b/TestPkg/LibraryClass.dsc
index 6530bafbe7..ee40055f07 100644
--- a/TestPkg/LibraryClass.dsc
+++ b/TestPkg/LibraryClass.dsc
@@ -111,6 +111,7 @@ NvVarsFileLib|TestPkg/Library/NvVarsFileLib/NvVarsFileLib.inf
 
 
 [LibraryClasses.common.DXE_CORE]
+  BaseMemoryLib|MdePkg/Library/BaseMemoryLibRepStr/BaseMemoryLibRepStr.ifn
   HobLib|MdePkg/Library/DxeCoreHobLib/DxeCoreHobLib.inf
   DxeCoreEntryPoint|MdePkg/Library/DxeCoreEntryPoint/DxeCoreEntryPoint.inf
   MemoryAllocationLib|MdeModulePkg/Library/DxeCoreMemoryAllocationLib/DxeCoreMemoryAllocationLib.inf
-- 
2.27.0.windows.1

